apps/web/app/(app)/gardens/[chain]/[garden]/[community]/page.tsx

```diff
@@ -0,0 +1,201 @@
+            <Statistic
+              label="registration amount"
+              count={registerStakeAmount}
+            />
+            <Statistic label="community fee" count={communityFee} />
+          </div> */}
+          <RegisterMember
+            tokenSymbol={tokenGarden?.symbol ?? ""}
+            communityAddress={communityAddr as Address}
+            registerToken={tokenAddr as Address}
+            registerTokenDecimals={tokenGarden?.decimals}
+            membershipAmount={registerStakeAmount}
+            protocolFee={protocolFee}
+            communityFee={communityFee}
+          />
+        </div>
+      </header>
+      <IncreasePower
+        communityAddress={communityAddr as Address}
+        registerToken={registerToken as Address}
+        tokenSymbol={tokenGarden?.symbol ?? ""}
+        registerTokenDecimals={tokenGarden?.decimals as number}
+        registerStakeAmount={BigInt(registerStakeAmount)}
+      />
+      <section className="section-layout flex flex-col gap-10">
+        <header className="flex justify-between">
+          <h2>Pools</h2>
+          <FormLink href={`/gardens/${chain}/${tokenAddr}/${communityAddr}/create-pool`} label="Create Pool" />
+        </header>
+        <div className="flex flex-col gap-4">
+          <h4 className="text-secondary-content">
+            Funding pools ( {fundingPools.length} )
+          </h4>
+          <div className="flex flex-row flex-wrap gap-10">
+            {fundingPools.map((pool) => (
+              <PoolCard
+                key={pool.poolId}
+                tokenGarden={tokenGarden as TokenGarden}
+                {...pool}
+              />
+            ))}
+          </div>
+        </div>
+        <div className="flex flex-col gap-4">
+          <h4 className="text-secondary-content">
+            Signaling pools ( {signalingPools.length} )
+          </h4>
+          <div className="flex flex-row flex-wrap gap-10">
+            {signalingPools.map((pool) => (
+              <PoolCard
+                key={pool.poolId}
+                tokenGarden={tokenGarden as TokenGarden}
+                {...pool}
+              />
+            ))}
+          </div>
+        </div>
+        <div className="flex flex-col gap-4">
+          <h4 className="text-secondary-content">
+            Pools in Review ( {poolsInReview.length} )
+          </h4>
+          <div className="flex flex-row flex-wrap gap-10">
+            {poolsInReview.map((pool) => (
+              <PoolCard
+                key={pool.poolId}
+                tokenGarden={tokenGarden as TokenGarden}
+                {...pool}
+              />
+            ))}
+          </div>
+        </div>
+      </section>
+      <section className="section-layout">
+        <h2 className="mb-4">Covenant</h2>
+        <p>{covenant}</p>
+        <div className="mt-10 flex justify-center">
+          <Image src={groupFlowers} alt="flowers" className="w-[265px]" />
```

Comment:
I think set the width as props is better for optimization:
            https://nextjs.org/docs/pages/api-reference/components/image#width